### PR TITLE
Grammar corrections

### PR DESCRIPTION
**Pull request by zraineri**
I just made some grammar corrections/verbiage updates to the first 3rd of the doc to help readability a bit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,14 @@ Monaco Editor for React
...
 
-There also exist solutions for integration with React; e.g [this one](https://github.com/react-monaco-editor/react-monaco-editor) and [this one](https://github.com/jaywcjlove/react-monacoeditor). But they need some custom webpack configuration to make Monaco fully work, which is not the "best" solution for such kind of things like create-react-app - [CRA](https://facebook.github.io/create-react-app/).
+There does already exist solutions for integration with React; e.g [this one](https://github.com/react-monaco-editor/react-monaco-editor) and [this one](https://github.com/jaywcjlove/react-monacoeditor). But they need custom webpack configuration for Monaco to fully work, which is not the "best" solution for things like create-react-app - [CRA](https://facebook.github.io/create-react-app/).
```
Comment (suren-atoyan) [inline README.md:11]: do we need `does` if `solutions` is plural?

**Comment by suren-atoyan** [comment]
Hello @zraineri 

I am really grateful for your corrections. All points you've mentioned are excellent; I just have one question (the comment is above) and I am not sure about that 😄 I am not a native speaker 🙂

File: README.md (hunk 1 above)
Comment (zraineri) [reply README.md:11]: We do not, I used it to accent the point almost as if answering their question "Do other solutions exist?" but it is 100% optional and not needed

**Comment by zraineri** [comment]
@suren-atoyan You're very well spoken for not being a native speaker! 🙂 Just glad to help, if you ever want me to review anything else just let me know! I love the library and really appreciate you putting it together/maintaining it. I'm really trying to be better about helping with the projects I use

**Comment by suren-atoyan** [comment]
@zraineri I am planning to release a new major version in the near future; there will be many API changes, as well as the whole documentation will be rewritten. So, I would be glad to share the new version with you and get feedback :)

I'll keep you up to date.

Thank you for your support! 🎉 🤝

**Comment by zraineri** [comment]
@suren-atoyan Can't wait 🎉 I'd love to help anyway I can 🤝

**Comment by suren-atoyan** [comment]
Hey @zraineri 👋 

The new version is here 🙂 

Here is the `monaco-react` with almost new README - [@monaco-editor/react](https://github.com/suren-atoyan/monaco-react)
And here is the `monaco-loader`; the `loader` utility became a separate library - [@monaco-editor/loader](https://github.com/suren-atoyan/monaco-loader)

I would love to get any feedback about them. Thank you!

**Pull request merged**